### PR TITLE
Fix mob fall in open space on zero gravity

### LINKED ISSUE #44679: fall into open space in zero gravity
fall into open space in zero gravity

## Reproduction:

walk up to 3 level of Multi Z debug where no grav gen and step to open space.

### PR DESCRIPTION
**Pull request by Dennok**
## About The Pull Request
fix#44679

## Why It's Good For The Game
Zero gravity is zero gravity. What push you to down in open space if there is no gravity?

## Changelog
:cl:
fix: Mob no more falling in zero gravity
/:cl:

### CODE REVIEW

**Comment by cacogen** [comment]
We had this years back but it was removed.

**Comment by Dennok** [comment]
Yes, this multi z open space falling.

File: code/modules/mob/living/living_movement.dm [hunk 1]
```diff
@@ -50,7 +50,7 @@
 	remove_movespeed_modifier(MOVESPEED_ID_PRONE_DRAGGING)
 
 /mob/living/can_zFall(turf/T, levels)
-	return !(movement_type & FLYING)
+	return ..() //!(movement_type & FLYING)
```
Comment (actioninja) [inline code/modules/mob/living/living_movement.dm]: why comment out the old code?

**Review by actioninja — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/living_movement.dm (hunk 1 above)
Comment (Dennok) [reply code/modules/mob/living/living_movement.dm]: atom/can_zFall hold all what need to make mob dont fall.
But it still need to be upgrade to hold mechanics from another dont standard sources like mutations, equipment, etc.
Just remove commented part ?

**Pull request merged**